Project.toml

```diff
@@ -4,9 +4,20 @@ authors = ["David Milovich <david.milovich@welkinsciences.com>"]
 version = "0.4.16"
 
 [deps]
-SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
-QuadGK = "1fd47b50-473d-5c70-9696-f719f8f3bcdc"
 Scratch = "6c6a2e73-6563-6170-7368-637461726353"
+SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
+
+[extras]
+QuadGK = "1fd47b50-473d-5c70-9696-f719f8f3bcdc"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+[targets]
+test = ["QuadGK", "Test"]
+
+[compat]
+HDF5 = "0.17"
+Scratch = "1"
+SpecialFunctions = "2"
+julia = "1.6"
```

Comment:
Similar to the test script of the gh actions, I propose to use '1.10' as the baseline, because it is the current LTS. Or is there any reason to support '1.6'?

Reply:
No reason on my end. I merely haven't yet tested FCCQuad.jl against more recent versions of Julia.